modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/seo.jsp

```diff
@@ -114,7 +114,103 @@ UnicodeProperties layoutTypeSettings = selLayout.getTypeSettingsProperties();
 			</div>
 		</c:otherwise>
 	</c:choose>
-	</></c:if>
+</c:if>
+
+<c:if test="<%= !StringUtil.equals(selLayout.getType(), LayoutConstants.TYPE_ASSET_DISPLAY) %>">
+	<h4><liferay-ui:message key="open-graph" /></h4>
+
+	<%
+	LayoutSEOEntry selLayoutSEOEntry = layoutsAdminDisplayContext.getSelLayoutSEOEntry();
+	%>
+
+	<c:choose>
+		<c:when test="<%= selLayoutSEOEntry != null %>">
+			<aui:model-context bean="<%= selLayoutSEOEntry %>" model="<%= LayoutSEOEntry.class %>" />
+
+			<div id="<portlet:namespace />openGraphSettings">
+				<aui:input checked="<%= selLayoutSEOEntry.isEnabledOpenGraphTitle() %>" helpMessage="use-custom-open-graph-title-help" id="useCustomTitle" label="use-custom-title" name="openGraphTitleEnabled" type="checkbox" />
+
+				<aui:input label="<%= StringPool.BLANK %>" name="openGraphTitle" placeholder="title" />
+
+				<aui:input checked="<%= selLayoutSEOEntry.isEnabledOpenGraphDescription() %>" helpMessage="use-custom-open-graph-description-help" id="useCustomDescription" label="use-custom-description" name="openGraphDescriptionEnabled" type="checkbox" />
+
+				<aui:input label="<%= StringPool.BLANK %>" name="openGraphDescription" placeholder="description" />
+
+				<aui:input id="openGraphImageFileEntryId" name="openGraphImageFileEntryId" type="hidden" />
+			</div>
+		</c:when>
+		<c:otherwise>
+			<div id="<portlet:namespace />openGraphSettings">
+				<aui:input checked="<%= false %>" helpMessage="use-custom-open-graph-title-help" id="useCustomTitle" label="use-custom-title" name="openGraphTitle" type="checkbox" />
+
+				<aui:input label="<%= StringPool.BLANK %>" localized="<%= true %>" name="openGraphTitle" type="text">
+				</aui:input>
```

Comment:
Can we remove the closing </aui:input>? All other inputs that have no value there are autoclosing.